packages/editor/src/components/inserter-sidebar/index.js

```diff
@@ -30,23 +30,14 @@ export default function InserterSidebar( {
 		const { getInserterSidebarToggleRef, getInsertionPoint } = unlock(
 			select( editorStore )
 		);
-		const { getBlockRootClientId, __unstableGetEditorMode, getSettings } =
-			select( blockEditorStore );
+		const { getSettings } = select( blockEditorStore );
 		const { get } = select( preferencesStore );
-		const getBlockSectionRootClientId = () => {
-			if ( __unstableGetEditorMode() === 'zoom-out' ) {
```

Comment:
We should always fall back to the section ID without checking zoom-out mode. Otherwise the inserter won't have any patterns, and it's impossible to enable zoom-out mode. It's not a bad thing to always fall back to section ID regardless of mode.